package.json

```diff
@@ -40,23 +40,34 @@
     "cli-progress": "^3.8.2",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
+    "crc": "^3.8.0",
     "encryptedfs": "^0.1.0",
     "fs-extra": "^9.0.0",
     "futoin-hkdf": "^1.3.2",
+    "git-list-pack": "0.0.10",
+    "http-parser-js": "^0.5.1",
     "inquirer": "^7.1.0",
     "inquirer-fuzzy-path": "^2.3.0",
-    "isomorphic-git": "^0.35.1",
+    "isomorphic-git": "^1.5.0",
     "jsonfile": "^6.0.1",
     "kbpgp": "^2.0.82",
+    "marky": "^1.2.1",
     "moniker": "^0.1.2",
     "node-fetch": "^2.6.0",
+    "node-forge": "^0.9.1",
+    "pad": "^3.2.0",
+    "simple-get": "^4.0.0",
+    "split-buffer": "^1.0.0",
+    "stream-source": "^0.3.5",
     "threads": "^1.6.0",
+    "url": "^0.11.0",
```

Comment:
Just to be clear, is this because we want to use Node URL but in other platforms? Maybe in that case, we should just have our own utility function.

Reply:
This has been replaced by a simple string split and regex for checking the url path.